docs/main/existing-rhdh.md

```diff
@@ -115,7 +115,7 @@ By default it should point to `http://sonataflow-platform-data-index-service.son
         pluginConfig:
           dynamicPlugins:
             frontend:
-              janus-idp.backstage-plugin-orchestrator:
+              red-hat-developer-hub.backstage-plugin-orchestrator:
```

Comment:
is this necessary for this PR?

Reply:
if this PR updates the plugins, it's necessary

Reply:
Will remove it from this PR since we want a separate PR for the orchestrator plugin updates.

Reply:
Done